CHANGELOG.md

```diff
@@ -1,11 +1,21 @@
 # Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [0.7.0]
+
+- Allow transaction with undefined 'to' fields, in signTransaction ([#98](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/98))
+- Update eth-ledger-bridge-keyring to support EIP-1559 transactions ([#97](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/97), [#96](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/96))
+- Update gh-pages to receive chainId, update Ledger library to 6.5.0 ([#95](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/95))
```

Comment:
This change was made to the bridge, not the keyring. Totally different branch even. This can be omitted as well.